modules/installation-generate-aws-user-infra-install-config.adoc

```diff
@@ -116,6 +123,22 @@ publish: Internal
 By setting this option, you create an internal Ingress Controller and a private load balancer.
 endif::restricted[]
 
+ifdef::localzone[]
+. Edit the `install-config.yaml` file to provide the subnets for the availability zones that your VPC uses:
++
+[source,yaml]
+----
+compute:
```

Comment:
Did they change the format of the `install-config.yaml` file? All of the reference files that we have separates the compute and control plane definitions into separate stanzas and treats them differently. It's been awhile, but the installer team asked that we promote using the installation program to generate the initial `install-config.yaml` file whenever possible to make sure that users are starting from a file that's compatible with their version, just in case.